ambry-network/src/main/java/com.github.ambry.network/Transmission.java

```diff
@@ -132,8 +132,7 @@ public SocketAddress getRemoteSocketAddress() {
   /**
    * Close the connection for the socket channel
    */
-  public abstract void close()
-      throws IOException;
```

Comment:
@nsivabalan I don't agree that is a "fix". It makes the function signatures inconsistent.